src/Notes/DeprecatedNotes.php

```diff
@@ -29,6 +41,15 @@ class WC_Admin_Note extends DeprecatedClassFacade {
 	 * @var string
 	 */
 	protected static $deprecated_in_version = '1.6.0';
+
+	/**
+	 * Note constructor. Loads note data.
+	 *
+	 * @param mixed $data Note data, object, or ID.
+	 */
+	public function __construct( $data = '' ) {
```

Comment:
Just in case, when I was testing this branch in an ephemeral site (following the steps I mentioned [here](https://github.com/woocommerce/woocommerce-admin/pull/5215#issuecomment-700930416)) with the addition of this code I'm seeing [this warning](https://d.pr/i/Qk1Zg7):
```
Warning: call_user_func_array() expects parameter 1 to be a valid callback, first array member is not a valid class name or object in /srv/users/user1f7641a7/apps/user1f7641a7/public/wp-content/plugins/woocommerce-admin/src/DeprecatedClassFacade.php on line 55
```
Is that expected?